platform/macos/jazzy.yml

```diff
@@ -16,6 +16,9 @@ framework_root: ../darwin/src
 custom_categories:
   - name: Guides
     children:
+      - Adding Points to a Map
+      - Runtime Styling
+      - Working with Mapbox Studio
```

Comment:
These guides are located in platform/ios/docs/guides/, so they aren’t available to the macOS SDK. While most of the Runtime Styling and Working with Mapbox Studio guides apply equally to the macOS SDK, some of the screenshots would need to be replaced, and the “Adding Points to a Map” guide discusses view-backed annotations, which have yet to be implemented on macOS: #5035.

Let’s leave these guides out for now. We can figure out how to bring them over to the macOS SDK in a separate PR. Maybe we should think at that point about whether any of this great content could be reused in Android SDK documentation as well.